src/components/application/FundingApplication.tsx

```diff
@@ -79,7 +81,12 @@ export const FundingApplication = ({
             throw new Error(result.error ?? "Error submitting application")
           }
 
-          resolve(result.applications[0])
+          const application = result.applications[0]
+          track("Apply", {
+            projectIds: application.projectId,
```

Comment:
should this just be `projectId`? also I guess we should loop and add an event for each project...